internal/repositories/equipment.go

```diff
@@ -564,6 +557,9 @@ func (r *equipmentRepository) BlockEquipment(
 		if err != nil {
 			return err
 		}
+	} else {
+		// Means multiple "not available" (blocking) statuses exist per single equipment. Must never happen
+		return err
```

Comment:
You can wrap it with fmt.Errorf("err message: %v", err ) for more verbose error
```suggestion
	} else {
		// Means multiple "not available" (blocking) statuses exist per single equipment. Must never happen
		return fmt.Errorf("multiple statuses not available for blocking period: %w", err)
```